card.py

```diff
@@ -0,0 +1,28 @@
+import env_vars as env
+from datetime import datetime
+
+class Card:
+    def __init__(self, id, idShort, title, due_date, is_complete):
+        self.id = id
+        self.idShort = idShort
+        self.title = title
+        self.due_date = due_date
+        self.is_complete = is_complete
+
+    @staticmethod
+    def is_done_list(id):
+        return id == env.DONE_LIST_ID
+
+    @staticmethod
```

Comment:
This might work better as a @classmethod (which takes the class as its first argument) as you would not have to repeat the class name twice in the below.